clients/src/main/java/org/apache/kafka/clients/consumer/RoundRobinAssignor.java

```diff
@@ -73,8 +73,8 @@
     }
 
 
-    public List<TopicPartition> allPartitionsSorted(Map<String, Integer> partitionsPerTopic,
-                                                    Map<String, Subscription> subscriptions) {
+    private List<TopicPartition> allPartitionsSorted(Map<String, Integer> partitionsPerTopic,
```

Comment:
Seems like this is a public API.

Reply:
It's a public method in a public class: https://kafka.apache.org/10/javadoc/?org/apache/kafka/clients/consumer/KafkaConsumer.html

I don't see what is the relevant of it being a helper function.